diff --git a/homework_notebook.Rmd b/homework_notebook.Rmd
index f26d85b..e09f293 100644
--- a/homework_notebook.Rmd
+++ b/homework_notebook.Rmd
@@ -27,10 +27,13 @@ library(tidyverse)
 Если это не так, исправьте это (может пригодиться функция `distinct`).
 
 ```{r}
+dm <- read_csv2("data/raw/mini/dm.csv")
 
-# выполняет user2
-# ветка dm
+dm <- dm %>%
+  select(USUBJID, SEX, ACTARM)
 
+dm <- dm %>%
+  distinct()
 ```
 
 ## 1.2. VS (Vital Signs)
