diff --git a/homework_notebook.Rmd b/homework_notebook.Rmd
index f26d85b..754131c 100644
--- a/homework_notebook.Rmd
+++ b/homework_notebook.Rmd
@@ -37,10 +37,13 @@ library(tidyverse)
 Выберите столбец `USUBJID` и столбец `VSORRES` под именем `TEMP`.
 
 ```{r}
+vs <- read_csv("data/raw/vs.csv")
 
-# выполняет user3
-# ветка vs
+vs <- vs %>%
+  filter(VSTESTCD == "TEMP", VISITNUM == 1)
 
+vs <- vs %>%
+  select(USUBJID, TEMP = VSORRES)
 ```
 
 Убедитесь, что в результате на каждого субъекта (`USUBJID`) приходится только одна строка.
