diff --git a/homework_notebook.qmd b/homework_notebook.qmd
index f26d85b..64015ab 100644
--- a/homework_notebook.qmd
+++ b/homework_notebook.qmd
@@ -53,10 +53,13 @@ library(tidyverse)
 Выберите столбец `USUBJID` и столбец `LBORRES` под именем `WBC`.
 
 ```{r}
+lb <- readxl::read_excel("data/raw/lb.xlsx")
 
-# выполняет user4
-# ветка lb
+lb <- lb |>
+  filter(LBTESTCD == "WBC", VISITNUM == 1)
 
+lb <- lb |>
+  select(USUBJID, WBC = LBORRES)
 ```
 
 Убедитесь, что теперь на каждого субъекта (`USUBJID`) приходится только одна строка.
