diff --git a/homework_notebook.Rmd b/homework_notebook.Rmd
index 42ecc70..2379de4 100644
--- a/homework_notebook.Rmd
+++ b/homework_notebook.Rmd
@@ -96,13 +96,8 @@ data <- data %>%
 
 ```{r}
 data <- data %>%
-<<<<<<< HEAD
   mutate(TEMP = as.numeric(str_replace(TEMP, ",", ".")),
-         WBC = WBC)
-=======
-  mutate(TEMP = TEMP,
          WBC = as.numeric(str_replace(WBC, ",", ".")))
->>>>>>> main
 ```
 
 # 4. Визуализация
@@ -113,17 +108,14 @@ data <- data %>%
 
 ```{r}
 
-<<<<<<< HEAD
 ggplot(data, aes(ACTARM, TEMP)) +
   geom_boxplot() +
   labs(y = "Temperature, C")
 
-=======
 ggplot(data, aes(ACTARM, WBC)) +
   geom_boxplot() +
   scale_y_log10() +
   labs(y = "WBC, 10^9/L")
->>>>>>> main
 
 ```
 
