diff --git a/homework_notebook.qmd b/homework_notebook.qmd
index 1f790ec..aa3b322 100644
--- a/homework_notebook.qmd
+++ b/homework_notebook.qmd
@@ -91,7 +91,7 @@ data <- data |>
 # перекодирование данных здесь
 
 data <- data |>
-  mutate()
+  mutate(WBC = as.numeric(str_replace(WBC, ",", ".")))
 ```
 
 # 4. Визуализация
@@ -105,9 +105,8 @@ data <- data |>
 ‼️ Перекодирование данных выполните в общем чанке в главе 3 выше.
 
 ```{r}
-# ветка boxplot-temp-sex
-# ветка boxplot-temp-actarm
-# ветка boxplot-wbc-actarm
+ggplot(data, aes(ACTARM, WBC)) +
+  geom_boxplot()
 ```
 
 # Дополнительные задания
