diff --git a/homework_notebook.Rmd b/homework_notebook.Rmd
index 1f790ec..6038988 100644
--- a/homework_notebook.Rmd
+++ b/homework_notebook.Rmd
@@ -97,7 +97,7 @@ data <- data %>%
 ```{r}
 data <- data %>%
   mutate(TEMP = TEMP,
-         WBC = WBC)
+         WBC = as.numeric(str_replace(WBC, ",", ".")))
 ```
 
 # 4. Визуализация
@@ -108,9 +108,10 @@ data <- data %>%
 
 ```{r}
 
-# ветка boxplot-temp-sex
-# ветка boxplot-temp-actarm
-# ветка boxplot-wbc-actarm
+ggplot(data, aes(ACTARM, WBC)) +
+  geom_boxplot() +
+  scale_y_log10() +
+  labs(y = "WBC, 10^9/L")
 
 ```
 
